### PR TITLE
small changes to the max-width

### PR DESCRIPTION
**Pull request by sturobson**

### CODE REVIEW

**Comment by khawkins98** [comment]
👍  

Also need a changleog for vf-design-tokens.

**Comment by khawkins98** [comment]
Saw a few other places 81.25 is coded in, also should be updated?

image

**Comment by sturobson** [comment]
Should be ready to go - I've not updated `grid-page` as it's deprecated (probably should be deleted from the codebase at some point?)

**Comment by khawkins98** [comment]
> probably should be deleted from the codebase at some point?

Good point on the archive code. Maybe just hide the component and add it to the .gitingnore so it stays out of our searches?

**Review by khawkins98 — APPROVED** [review]

File: components/vf-body/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+### 1.2.0
```
Comment (khawkins98) [inline components/vf-body/CHANGELOG.md]: Just noticed. should be 1.1.0?

**Review by khawkins98 — APPROVED** [review]

**Pull request merged**